app/cdap/styles/common.scss

```diff
@@ -49,6 +49,8 @@ html {
   position: relative;
   min-height: 100%;
   overflow-y: hidden;
+  max-width: 100vw;
+  overflow: hidden;
```

Comment:
This will affect all pages in the app - is it necessary or can it be scoped to Wrangler?

Reply:
yes, this is necessary, if any component is greater than the width of the page it is overflowing

we saw issues while implementing the horizontal expandable tabs component.
This is the only fix that could work.